### PR TITLE
Release 1.0.0

### PR DESCRIPTION
**Pull request by alexshoe**
Update to version 1.0.0 for the recently merged changes, bringing `dash-player` up to a production-ready state

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+# Change Log for Dash Player
+
+All notable changes to `dash-player` will be documented in this file.
+This project adheres to [Semantic Versioning](https://semver.org/).
+
+---
+
+## [1.0.0] - 2022-10-20
+
+This release is `v1.0.0` as it that `dash-player` is now production-ready.
```
Comment (alexcjohnson) [inline CHANGELOG.md:10]:
```suggestion
This release is `v1.0.0` to indicate that `dash-player` is now considered production-ready.
```

**Review by alexcjohnson — APPROVED** [review]
💃 after my one comment!

**Comment by antoinerg** [comment]
Congratulations on pushing this through @alexshoe!

**Comment by alexshoe** [comment]
Thanks @antoinerg! 😊🎉

**Comment by ndrezn** [comment]
Ditto here! Well done @alexshoe 🚀 🥳

**Pull request merged**